assets/queries/openAPI/security_operations_empty_object_definition/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "663c442d-f918-4f62-b096-0bf5dcbeb655",
+  "queryName": "Operations security field has an empty object",
```

Comment:
Maybe it would be better if every word of 'queryName' starts with uppercase

Reply:
The folder name is different from the 'queryName' in the metadata file

Reply:
Regarding the query name uppercase we cannot change it due to the marshall/unmarshall of the vulnerability struct. The second point regarding the name of the directory is valid